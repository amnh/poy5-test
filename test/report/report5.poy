(* Build report trees with parsimony branches; dynamic *)
set(seed:50, timer:0)
read ("1.fas")
build (2)
report (trees:(branches:min))
report (trees:(branches:max))
report (trees:(branches:single))
