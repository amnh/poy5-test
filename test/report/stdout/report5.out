(t20:12.5,((t11:36,t6:49):22,(t18:19,(t26:11,((t38:11,t47:13):13,((t33:3
,t40:5):12,(t27:19,((t28:30,(t1:23,(t25:2,t49:6):10):6):8,((t41:5,(t19:7
,t2:7):5):5,((t14:2,t3:9):6,((t9:0,(t8:0,(t32:2,((t29:5,t30:10):8,(t4:1
,t5:0):3):2):1):0):4,((t43:0,(t44:1,t45:0):1):9,(t48:4,(t22:0,((t24:0,(t23:0
,(t21:1,(t13:0,t37:0):1):2):0):1,(((t35:0,t36:0):3,(t15:2,(t16:1,t7:2):1):2
):0,(t12:10,(t31:5,(t34:146,(t46:6,(t10:9,(t39:4,(t17:2,t42:2):3):8):6):8):7
):5):6):2):1):3):4):7):4):4):10):11):13):10):15):10):17):12.5)
;

(t20:27,((t11:84,t6:81):98,(t18:35,(t26:30,((t38:27,t47:29):39,((t33:16
,t40:16):30,(t27:32,((t28:37,(t1:35,(t25:14,t49:20):39):62):49,((t41:14
,(t19:16,t2:17):23):24,((t14:14,t3:14):20,((t9:4,(t8:1,(t32:5,((t29:19
,t30:20):20,(t4:4,t5:4):12):14):5):5):12,((t43:10,(t44:2,t45:2):11):18,(t48:8
,(t22:4,((t24:1,(t23:2,(t21:2,(t13:1,t37:1):2):2):2):4,(((t35:3,t36:3):4
,(t15:4,(t16:4,t7:4):7):6):10,(t12:17,(t31:19,(t34:153,(t46:21,(t10:20
,(t39:11,(t17:5,t42:8):12):23):28):158):156):24):18):11):7):9):21):21):16):22
):29):38):42):39):39):47):59):27)
;

(t38:5.5,(t47:12,(((t11:33,t6:55):25,(t18:21,(t20:24,t26:8):9):15):14,(t27:13
,(t28:30,(((t25:3,t49:5):11,(t33:3,t40:5):17):8,((t41:7,(t19:7,t2:7):5):4
,((t8:0,(t9:0,(t32:2,((t4:1,t5:0):0,(t44:1,(t43:1,t45:1):1):13):3):1):0):3
,((t29:3,t30:10):9,((t14:2,t3:9):5,((t1:24,(t12:8,(t31:5,(t34:147,(t46:6
,(t10:9,(t17:2,(t39:4,t42:2):2):8):7):9):6):7):5):4,((t35:0,t36:0):3,(t15:2
,((t16:1,t7:2):1,((t22:0,t48:4):1,(t23:0,(t24:0,(t37:0,(t13:0,t21:1):0):2):0
):2):1):1):2):4):7):2):1):7):12):11):11):14):11):5.5)
;

(t38:13.5,(t47:28,(((t11:87,t6:86):95,(t18:38,(t20:36,t26:36):55):53):55
,(t27:33,(t28:46,(((t25:17,t49:23):31,(t33:21,t40:19):30):38,((t41:16,(t19:16
,t2:17):22):29,((t8:3,(t9:1,(t32:6,((t4:1,t5:1):18,(t44:14,(t43:2,t45:2):14
):17):19):6):4):9,((t29:19,t30:21):21,((t14:13,t3:14):18,((t1:32,(t12:15
,(t31:19,(t34:153,(t46:20,(t10:21,(t17:9,(t39:6,t42:9):12):22):29):158):156
):21):38):38,((t35:3,t36:3):9,(t15:4,((t16:4,t7:4):6,((t22:5,t48:5):8,(t23:2
,(t24:2,(t37:2,(t13:1,t21:1):2):2):3):3):4):6):10):17):15):18):17):19):29):49
):60):49):42):13.5)
;

(t20:5,((t11:28,t6:36):22,(t18:19,(t26:4,((t38:11,t47:6):7,((t33:2,t40:3):10
,(t27:13,((t28:28,(t1:22,(t25:0,t49:6):8):2):4,((t41:4,(t19:5,t2:7):4):3
,((t14:0,t3:9):4,((t9:0,(t8:0,(t32:2,((t29:4,t30:7):8,(t4:1,t5:0):1):2):1):0
):0,((t43:0,(t44:1,t45:0):1):8,(t48:3,(t22:0,((t24:0,(t23:0,(t21:0,(t13:0
,t37:0):1):2):0):1,(((t35:0,t36:0):2,(t15:1,(t16:1,t7:2):2):2):0,(t12:8
,(t31:1,(t34:142,(t46:1,(t10:8,(t39:2,(t17:1,t42:1):2):6):6):6):7):4):5):2):1
):1):2):7):3):3):5):7):7):5):13):3):13):5)
;
(t38:5.5,(t47:7,(((t11:23,t6:41):19,(t18:12,(t20:22,t26:4):8):6):10,(t27:9
,(t28:25,(((t25:3,t49:3):6,(t33:2,t40:3):13):3,((t41:4,(t19:5,t2:7):4):2
,((t8:0,(t9:0,(t32:2,((t4:1,t5:0):0,(t44:0,(t43:1,t45:0):1):13):3):1):0):2
,((t29:1,t30:10):7,((t14:1,t3:8):3,((t1:22,(t12:8,(t31:1,(t34:142,(t46:1
,(t10:8,(t17:1,(t39:3,t42:1):1):6):6):6):7):4):5):2,((t35:0,t36:0):3,(t15:1
,((t16:1,t7:2):1,((t22:0,t48:4):0,(t23:0,(t24:0,(t37:0,(t13:0,t21:1):0):2):0
):2):1):1):1):3):7):2):0):5):7):8):9):9):9):5.5)
;

